mne/io/base.py

```diff
@@ -1242,10 +1242,29 @@ def filter(self, l_freq, h_freq, picks=None, filter_length='auto',
                 logger.info('Filtering a subset of channels. The highpass and '
                             'lowpass values in the measurement info will not '
                             'be updated.')
-        filter_data(self._data, self.info['sfreq'], l_freq, h_freq, picks,
-                    filter_length, l_trans_bandwidth, h_trans_bandwidth,
-                    n_jobs, method, iir_params, copy=False, phase=phase,
-                    fir_window=fir_window, fir_design=fir_design)
+        # Deal with annotations
+        if skip_by_annotation is None:
+            if self.annotations is not None and any(
+                    desc.upper().startswith('EDGE')
+                    for desc in self.annotations.description):
+                warn('skip_by_annotation defaults to [] in 0.15 but will '
```

Comment:
it is `None` in the list of params but the behavior is equivalent to that of `[]` in 0.15, so this is how we sometimes describe the triaging